packer/scripts/firefox.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -eu
+
+FIREFOX_VERSION="81.0b2`
+
+cd /opt && \
+  curl -O https://download-installer.cdn.mozilla.net/pub/devedition/releases/${FIREFOX_VERSION}/linux-x86_64/en-US/firefox-${FIREFOX_VERSION}.tar.bz2 && \
+  tar -xf firefox-81.0b2.tar.bz2 && \
+  rm -rf firefox-81.0b2.tar.bz2
```

Comment:
These should use the `FIREFOX_VERSION` environment variable too.

Or, we could do a wildcard, e.g., `firefox-*.tar.bz2`

Reply:
ah right.

Took it out since Vagrant had issues
`==> pentesting-vagrant.vagrant.alpine: /tmp/script_1871.sh: line 10: syntax error: EOF in backquote substitution`

Reply:
alright I fixed it but I notice that firefox isn't in the vagrant box, so working on fixing that.